src/components/dialogs/parameters/network-visualizations/map-parameters.tsx

```diff
@@ -49,23 +49,21 @@ export const MapParameters = () => {
         name: `${TabValue.MAP}.${PARAM_LINE_FLOW_COLOR_MODE}`,
     });
 
-    const onSliderChange = (value: any) => {
-        setValue(`${TabValue.MAP}.${PARAM_LINE_FLOW_ALERT_THRESHOLD}`, Number(value), {
-            shouldValidate: true,
-        });
+    const onSliderChange = (value: number) => {
+        const dirty = { shouldDirty: true };
+        setValue(`${TabValue.MAP}.${PARAM_LINE_FLOW_ALERT_THRESHOLD}`, value, dirty);
     };
-
     // the translation of values
     const lineFlowModeOptions = useMemo(() => {
-        return intlLineFlowModeOptions();
+        return INTL_LINE_FLOW_MODE_OPTIONS;
     }, []);
 
     const lineFlowColorModeOptions = useMemo(() => {
-        return intlLineFlowColorModeOptions();
+        return INTL_LINE_FLOW_COLOR_MODE_OPTIONS;
     }, []);
 
     const mapBaseMapOptions = useMemo(() => {
-        return intlMapBaseMapOptions();
+        return INTL_MAP_BASE_MAP_OPTIONS;
```

Comment:
useMemo is not usefull when you return a constant

Reply:
Indeed. I forgot to remove it